### DISCUSSION TITLE
Help :)

### DISCUSSION THREAD

**Discussion by digimumo** [cjmellor/level-up#60] [General]
Hello, I've installed the level up package and trying to use addPoints() but it spits out a SQL error relating to foreign keys. 

SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`xxxxxxxx`.`experiences`, CONSTRAINT `experiences_level_id_foreign` FOREIGN KEY (`level_id`) REFERENCES `levels` (`id`)).

I've also setup the levels in the levels table 1 => 100 2=>200 etc. 

What am I doing wrong here? thanks

**Comment by cjmellor** [comment]
Can you list out the steps you took to get to this error?

Thanks
